Park-Swift/App.js

```diff
@@ -12,6 +12,7 @@ import { NavigationContainer } from '@react-navigation/native';
 import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs';
 import CurrentTile from './screens/home';
 import SearchBar from './screens/search';
+import RVP from './screens/recentlyVisitedProfiles.js';
```

Comment:
Does this need to be RecentlyVisitedProfiles written out instead of just RVP?

Reply:
I named it RVP intentionally to shorten it up, but I can expand the name if that's more conventional. Let me know what you prefer!

Reply:
@jennweir

Reply:
I like the idea of efficiency in general but good naming convention for React components is filename: MyComponent.js and the actual component itself being MyComponent so it can then be imported and used as <MyComponent>. I know no everyone is following the file naming at this point but I prefer long names that are more descriptive than acronyms for generic readability.

Reply:
sounds good! I will make that edit and commit the changes @jennweir